test/WebSites/FiltersWebSite/Filters/AuthorizationFilterAttribute.cs

```diff
@@ -1,13 +1,15 @@
-// Copyright (c) .NET Foundation. All rights reserved.
+﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
 using System.Linq;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.Authorization;
+using Microsoft.AspNetCore.Mvc.Filters;
 using Microsoft.AspNetCore.Mvc.Internal;
 
-namespace Microsoft.AspNetCore.Mvc.Filters
+namespace FiltersWebSite
 {
     [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = true, Inherited = true)]
     public abstract class AuthorizationFilterAttribute :
```

Comment:
Following are some references of this attribute: https://github.com/aspnet/Mvc/search?utf8=%E2%9C%93&q=authorizationfilterattribute